python/doc/examples/probabilistic_modeling/probabilistic_modeling.rst

```diff
@@ -54,6 +54,7 @@ Stochastic processes
     white_noise_process.ipynb
     random_walk_process.ipynb
     functional_basis_process.ipynb
+    add_trend.ipynb
     trend_transform.ipynb
```

Comment:
The TrendTransform class enables to create a new process Y from a process X : Y(\omega, t) = X(\omega, t) + f(t)
No hypothesis on X needed.
If f is estimated from a realization of Y, the function f is such that X is centered: E[X(\omega, t)]=0 for all t;

I propose to erase the sentence 'For example it may be useful to get the stationary fiedl...'.
The script is a little bit confused... I propose to simplify it with the following actions: 
* block 1: create a process X(\omega, t), one realization _fieldX of X and a function f
* block 2: add it to the process to create the process Y(\omega, t) = X(\omega, t) + f(t)
* block 3: add it to the field fieldX to create one field of Y:  fieldY  
* block 4: remove it from the Y process and from the field fieldY
* block 5 : estimate f from a functional basis and one realisation of Y